doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.rst

```diff
@@ -157,15 +155,15 @@ To build the described example with the provided manifest templates taken from y
 
         .. code-block:: console
 
-            west build -d C:/ncs-lcs/work_dir/build/ -b nrf54h20dk/nrf54h20/cpuapp -p -- -DSB_CONFIG_SUIT_ENVELOPE_EDITABLE_TEMPLATES_LOCATION="c:/my_templates" -DSB_CONFIG_SUIT_ENVELOPE_ROOT_TEMPLATE="c:/my_default_templates/root.yaml.jinja2" -DCONFIG_SUIT_ENVELOPE_APP_TEMPLATE="c:/my_default_templates/app.yaml.jinja2" -DCONFIG_SUIT_ENVELOPE_HCI_RPMSG_SUBIMAGE_TEMPLATE="c:/my_default_templates/radio.yaml.jinja2"
+            west build --sysbuild -d C:/ncs-lcs/work_dir/build/ -b nrf54h20dk/nrf54h20/cpuapp -p -- -DSB_CONFIG_SUIT_ENVELOPE_EDITABLE_TEMPLATES_LOCATION="c:/my_templates" -DSB_CONFIG_SUIT_ENVELOPE_ROOT_TEMPLATE="c:/my_default_templates/root.yaml.jinja2" -DCONFIG_SUIT_ENVELOPE_TEMPLATE="c:/my_default_templates/app.yaml.jinja2" -Dhci_ipc_CONFIG_SUIT_ENVELOPE_SUIT_ENVELOPE_TEMPLATE="c:/my_default_templates/radio.yaml.jinja2"
```

Comment:
Here as well as for the other examples - how do we escape the paths for `CONFIG_SUIT_ENVELOPE_TEMPLATE` (and other paths) on Windows? This needs to be tested